lib/cron/cache-pools-ipfs.ts

```diff
@@ -0,0 +1,112 @@
+import pinataSDK from '@pinata/sdk';
+import { ChainId, SubgraphProvider } from '@uniswap/smart-order-router';
+import { EventBridgeEvent, ScheduledHandler } from 'aws-lambda';
+import { Route53, STS } from 'aws-sdk';
+import { default as bunyan, default as Logger } from 'bunyan';
+import fs from 'fs';
+
+// if (process.env.STAGE == 'beta') {}
+
+const DIRECTORY = '/tmp/pools/';
+
+// add more chains here
+const chains = [ChainId.MAINNET, ChainId.RINKEBY]
+const names = ["mainnet.txt", "rinkeby.txt"]
```

Comment:
Can we remove the `.txt`? ideally the ipns is simply like `beta.api.uniswap.org/v1/pools/mainnet`

Reply:
Oh also we need a way to specify v3 vs v2 protocol.. `pools/mainnet/v3`?

Reply:
Hm, so I can't remove .txt because I need to write to a file otherwise I get an error about writing to a directory. Can be a different extension though

Reply:
Hmm ok I think .json makes more sense then

Reply:
Ok sounds good - updated with `ipfs.io/ipns/api.uniswap.org/v1/pools/v3/mainnet.json` but lmk if we should change it